VMEncryption/main/Utils/WAAgentUtil.py

```diff
@@ -27,6 +27,9 @@
 #
 def searchWAAgent():
     agentPath = '/usr/sbin/waagent'
+    if os.path.isfile(agentPath):
+        return agentPath
+    agentPath = '/usr/share/oem/bin/waagent'
```

Comment:
If an other unsupported operating system or images are introduced that have yet another agent path outside of these two, it would be good if this logic explicitly failed with telemetry exit code indicating an unsupported environment after attempting all known agent paths.